modules/dxp/apps/analytics/analytics-reports-js-components-web/package.json

```diff
@@ -0,0 +1,15 @@
+{
+	"dependencies": {
+		"react": "16.12.0"
+	},
+	"main": "js/index.js",
+	"name": "analytics-reports-js-components-web",
```

Comment:
nit: rename this to be prefixed with `@liferay/`

Reply:
Done